tsconfig.json

```
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
+      "target": "es2015",
+      "module": "commonjs",
+      "moduleResolution": "node",
+      "declaration": true,
+      "outDir": "./lib",
+      "strict": false ,
```

Given this is a new package, I would advocate for setting this to true. Did you run into problems doing that?

```suggestion
      "strict": true ,
```

---

Done